torch/ao/quantization/observer.py

```diff
@@ -349,6 +351,7 @@ class MinMaxObserver(_ObserverBase):
         quant_min: Minimum quantization value. If unspecified, it will follow the 8-bit setup.
         quant_max: Maximum quantization value. If unspecified, it will follow the 8-bit setup.
         eps: Epsilon value for float32, Defaults to `torch.finfo(torch.float32).eps`.
+        is_dynamic: Indicator that dynamic quantization will be used for observed module.
```

Comment:
maybe we can give a more detailed docstring for this flag, e.g. specifically mentioning how this flag impacts the conversion from observer to quantize op.
(1). when is_dynamic is False, the observer will be converted to quantize_per_tensor - dequantize ops in convert
(2). when is_dynamic is True, the observer will be converted to quantize_per_tensor_dynamic - dequantize ops